dockerfiles/Dockerfile

```diff
@@ -8,7 +8,7 @@ RUN apt update && apt install -y make python gcc
 RUN apt-get update && apt-get install -y ca-certificates
 ENV NPM_CONFIG_PREFIX=/home/node/.npm-global
 ENV PATH=$PATH:/home/node/.npm-global/bin
-RUN npm install --global snyk-broker@4.156.1
+RUN npm install --global snyk-broker
```

Comment:
I presume you also checked that the vuln is gone too?

Reply:
Tough cookie is completely dropped since version `4.157.3` 
https://github.com/snyk/broker/pull/570